hardhat/dist/hardhat.config.js

```diff
@@ -62,9 +62,16 @@ exports.unlockSigner = unlockSigner;
             process.exit(1);
     }
 });
-const forking = {
-    url: constants_1.ETHEREUM_RPC_URL,
-    blockNumber: constants_1.FORKED_BLOCK,
+const sharedNetworkSettings = {
+    live: false,
+    allowUnlimitedContractSize: true,
+    gas: 'auto',
+    gasPrice: 'auto',
+    gasMultiplier: 1,
+    accounts: {
+        mnemonic: 'coordinape',
```

Comment:
are these account ever used? I think they get some ETH prefilled when the network spins up. Seems like this will be a different address than LOCAL_SEED_ADDRESS or HARDHAT_OWNER_ADDRESS ...

It would be nice if the whatever is the devs primary eth address has some ETH for txs automatically

Reply:
yeah, that's what HARDHAT_OWNER_ADDRESS does